src/tools/crawler/protocol.rs

```diff
@@ -16,11 +16,13 @@ use ziggurat::{
 };
 
 use super::network::KnownNetwork;
+use crate::network::NodeState;
 
 pub const NUM_CONN_ATTEMPTS_PERIODIC: usize = 500;
-pub const MAX_CONCURRENT_CONNECTIONS: u16 = 1000;
-pub const MAIN_LOOP_INTERVAL: u64 = 5;
+pub const MAX_CONCURRENT_CONNECTIONS: u16 = 1200;
+pub const MAIN_LOOP_INTERVAL: u64 = 20;
 pub const RECONNECT_INTERVAL: u64 = 5 * 60;
+pub const MAX_WAIT_FOR_ADDR: u64 = 3 * 60;
```

Comment:
For time duration constants, I put the unit in the name of the const.  E.g., `MAIN_LOOP_INTERVAL_SECS`.
Then, I don't have to think about what the scalar applies to.

Reply:
Done.